src/monmove.cpp

```diff
@@ -1442,3 +1575,35 @@ int monster::turns_to_reach( int x, int y )
 
     return int( turns + .9 ); // Halve (to get turns) and round up
 }
+
+std::pair<item *, tripoint> monster::get_most_desired_item_in_radius( int radius,
+        std::function<float( const item & )> calc_desirability ) const
+{
+    //debugmsg( "Looking for items in radius %d", radius );
```

Comment:
There should be no commented out code. It has no meaning and is just clutter. It's fine for your working branch, but at some point it should get removed.